src/hooks/useLegacyUIURL.ts

```diff
@@ -25,6 +25,7 @@ export const useLegacyUIURL = (): string | null => {
       [routes.spawnHost]: `${uiURL}/spawn#?resourcetype=hosts`,
       [routes.spawnVolume]: `${uiURL}/spawn#?resourcetype=volumes`,
       [`${routes.commits}/:id`]: `${uiURL}/waterfall/${id}`,
+      [`${routes.distroSettings}/*`]: `${uiURL}/distros##${id}`,
```

Comment:
Not sure if I'm doing something wrong but the link for this is always just this for me: 

<img width="248" alt="Screenshot 2023-10-06 at 4 42 44 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/8df0da68-cda9-41b0-a0b9-b1052da6517e">